web/html/src/manager/minion/ansible/schedule-playbook.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { render, server, screen } from "utils/test-utils";
+import { rest } from 'msw';
+import selectEvent from 'react-select-event';
+import { JsonResult } from "utils/network";
+import { AnsiblePath } from "./ansible-path-type";
+import SchedulePlaybook from "./schedule-playbook";
+import { PlaybookDetails } from "./accordion-path-content";
+
+const API_INVENTORIES = "/rhn/manager/api/systems/details/ansible/paths/inventory/1000";
+const API_PLAYBOOK_CONTENT = "/rhn/manager/api/systems/details/ansible/paths/playbook-contents";
+
+// TODO: Handle "ReferenceError: handleSst is not defined"
+window.handleSst = undefined;
+
+describe("Ansible playbooks", () => {
+  test("Render playbook content", async () => {
+    // Mock inventory list API endpoint
+    const response: JsonResult<AnsiblePath[]> = {
+      success: true,
+      messages: [],
+      data: [
+        {
+          id: 1,
+          minionServerId: 1000,
+          type: "inventory",
+          path: "/my/inventory"
+        },
+        {
+          id: 2,
+          minionServerId: 1000,
+          type: "inventory",
+          path: "/my/second/inventory"
+        },
+      ]
+    };
+    server.mockGetJson(API_INVENTORIES, response);
+
+    // Mock playbook content API endpoint
+    server.use(
+      rest.post(API_PLAYBOOK_CONTENT, (req, res, ctx) =>
+        res(ctx.json({
+          success: true,
+          messages: [],
+          data: "My playbook content string"
+        }))
+      )
+    );
+
+    // TODO: Setup automatically with datetimepicker
```

Comment:
I'm not entirely sure what you mean here, could you elaborate please?

Reply:
I mean the mocking of `/rhn/manager/api/maintenance/upcoming-windows` endpoint below. It's required because it's part of the `ActionSchedule` component so it's not directly related to these tests. I think this mock can be done in a more general place whenever `ActionSchedule` component is involved.

Maybe @parlt91 has some insights

Reply:
At the moment, I couldn't find what the problem is if this request isn't mocked, perhaps we can discuss in depth when you're back on Monday.

Reply:
Weird.. now I can't see the problem anymore either. Perhaps it is somehow gone with a rebase. So we can ignore this one.